core/documents-manager.php

```diff
@@ -275,7 +275,10 @@ public function get_doc_or_auto_save( $id, $user_id = 0 ) {
 	 * @return false|Document The document if it exist, False otherwise.
 	 */
 	public function get_doc_for_frontend( $post_id ) {
-		if ( is_preview() || Plugin::$instance->preview->is_preview_mode() ) {
+		$is_preview = is_preview() && $post_id === (int) Utils::get_super_global_value( $_GET, 'preview_id' );
+		$is_nonce_verify = wp_verify_nonce( Utils::get_super_global_value( $_GET, 'preview_nonce' ), 'post_preview_' . (int) Utils::get_super_global_value( $_GET, 'preview_id' ) );
```

Comment:
```suggestion
		$preview_id = (int) Utils::get_super_global_value( $_GET, 'preview_id' );
		$is_preview = is_preview() && $post_id === $preview_id;
		$is_nonce_verify = wp_verify_nonce( Utils::get_super_global_value( $_GET, 'preview_nonce' ), 'post_preview_' . $preview_id;
```
Sorry one more. You can cache (int) Utils::get_super_global_value( $_GET, 'preview_id' ); in a variable.

Reply:
fixed